src/components/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import { TodoItem } from "./TodoItem";
+import { useTodosContext } from "../context/TodosContext";
+import { FilterType } from "../types/todoTypes";
+import { useFilterContext } from "../context/FilterContext";
+
+export function TodoList() {
+  const { todos } = useTodosContext();
+  const { currentFilter } = useFilterContext();
+
+  const sortedTodos = todos.filter((todo) => {
+    if (currentFilter === FilterType.Completed) {
+      return todo.completed;
+    } else if (currentFilter === FilterType.Active) {
+      return !todo.completed;
+    }
+    return true;
+  });
```

Comment:
this utility also belongs to Context, you can pass `currentFilter` to function exported from context, you can also use useFilterContext hooks inside of useTodosContext, if available

Reply:
I moved it to the todo context, where I import filters.